pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -298,7 +298,7 @@ func TestPriorityQueue_AddUnschedulableIfNotPresent(t *testing.T) {
 // Pods in and before current scheduling cycle will be put back to activeQueue
 // if we were trying to schedule them when we received move request.
 func TestPriorityQueue_AddUnschedulableIfNotPresent_Backoff(t *testing.T) {
-	q := NewPriorityQueueWithClock(nil, clock.NewFakeClock(time.Now()), nil)
+	q := NewPriorityQueue(nil, nil, WithClock(clock.NewFakeClock(time.Now())))
```

Comment:
can we add tests for creating a priority queue WithPodInitialBackoffDuration and WithPodMaxBackoffDuration?